docs/registry/access-mgmt-registry/README.adoc

```diff
@@ -100,10 +100,11 @@ An account in {product-long-registry} is either a user account or a service acco
 
 The {registry} web console provides an *Access* tab on the {registry} instance page. {registry} instance owners, instance administrators, and organization administrators can use this tab to manage the following user roles:
 
-* *Administrator* - Can perform the following in this {registry} instance:
-** View or write user roles
+* *Administrator* - Can perform the following tasks in this {registry} instance:
```

Comment:
```suggestion
* *Administrator* 
Users with the Administrator role can perform the following tasks in this {registry} instance:
```

I might not have coded it correctly, but basically I'm suggesting using a term definition list, where the term is on a line of its own to make it easier to read.

Also rephrased to use a complete sentence.

Reply:
Sure, I think you mean a description list? https://docs.asciidoctor.org/asciidoc/latest/lists/description/

So format is `Administrator:: Users with the Administrator role...`

Reply:
Perfect, thanks! It's called a _definition list_ in IBM Style, see the Lists topic.